docs/shared/recipes/add-stack/add-solid.md

```diff
@@ -2,7 +2,7 @@
 
 The code for this example is available on GitHub:
 
-{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/solidjs" %}
+{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/solidjs" /%}
```

Comment:
Why do we have the backslash only here? Why is this needed?

Reply:
If the backslash isn't there, the entire rest of the article is hidden.  I'm not sure the technical reason why that happens.